napari/_qt/qt_main_window.py

```diff
@@ -49,7 +50,10 @@ def __init__(self, qt_viewer, *, show=True):
         self._qt_window.setCentralWidget(self._qt_center)
         self._qt_window.setWindowTitle(self.qt_viewer.viewer.title)
         self._qt_center.setLayout(QHBoxLayout())
-        self._status_bar = self._qt_window.statusBar()
+        # self._status_bar = self._qt_window.statusBar()
+        # if you guys approve the change i will clean this line
```

Comment:
Can you remove these comments (a short followup PR will do, I can quickly merge)

Reply:
i was thinking making an issue and referring there but i can remove since now we all following that discussion. let me make a small follow up PR.